pyproject.toml

```diff
@@ -0,0 +1,37 @@
+[tool.poetry]
```

Comment:
Now that you've added poetry, we no longer need to use pip. You should delete the requirements.txt file and update the README with instructions to install poetry / use `poetry run flask run` etc.

Also it's worth mentioning in there that the app can run in Vagrant.